Dockerfile

```diff
@@ -3,7 +3,11 @@ FROM node:8
 # Create app directory
 WORKDIR /usr/src/app
 
-RUN apt-get update && apt-get -y upgrade
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+  graphicsmagick \
```

Comment:
Let's try to keep away from a requirement to install packages into the actual host OS.